docs/.vitepress/theme/style.css

```
@@ -1,7 +1,6 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss" important;
```

ESLint

[error] Replace `"tailwindcss"` with `'tailwindcss'`

(format/prettier)